### PR TITLE
Solves #8(Created pull_request_template.md)

### PR DESCRIPTION
**Pull request by Archies11**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Description
+
+Please include a brief description of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Solves # (issue)
```
Comment (ananya-agrawal) [inline .github/pull_request_template.md]: change solves to fixes

**Comment by ananya-agrawal** [comment]
@pradeepgangwar review

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
```
Comment (pradeepgangwar) [inline .github/pull_request_template.md]: Remove this section.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Type of change
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
```
Comment (pradeepgangwar) [inline .github/pull_request_template.md]: We can remove this since there are no tests.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* SDK:
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (pradeepgangwar) [inline .github/pull_request_template.md]: Remove all lines below this.

**Comment by Archies11** [comment]
I have made the necessary changes.

**Comment by pradeepgangwar** [comment]
@ananya-agrawal LGTM now.

**Pull request merged**